src/Sarif/Autogenerated/FileChange.cs

```diff
@@ -34,11 +34,11 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// The location of the file to change.
         /// </summary>
-        [DataMember(Name = "fileLocation", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "fileLocation", IsRequired = true)]
         public FileLocation FileLocation { get; set; }
 
         /// <summary>
-        /// An array of replacement objects, each of which represents the replacement of a single range of bytes in a single file specified by 'fileLocation'.
+        /// An array of replacement objects, each of which represents the replacement of a single range of bytes in a single file specified by 'uri'.
```

Comment:
>uri [](start = 144, length = 3)

The OASIS schema is wrong and the SDK's version is right. The property is named `fileLocation`, not `uri`.